pkg/stable.packages.props

```diff
@@ -0,0 +1,950 @@
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.16.04-x64.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-arm.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x64.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x86.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x64.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x86.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win8-x64.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win8-x86.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win81-x64.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win81-x86.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win10-arm-aot.runtime.native.System.IO.Compression">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win10-x64-aot.runtime.native.System.IO.Compression">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win10-x86-aot.runtime.native.System.IO.Compression">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win8-arm.Microsoft.NETCore.Runtime.CoreCLR">
+      <Version>1.0.2</Version>
+    </StablePackage>
+
+    <!-- NETCore 1.0 WCF -->
+    <StablePackage Include="System.Private.ServiceModel">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Duplex">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Http">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.NetTcp">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Primitives">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Security">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    
+    <!-- NETCore 1.0 CoreCLR -->
+    <StablePackage Include="Microsoft.NETCore.Runtime.CoreCLR">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    
```

Comment:
We may be able to remove everything above here because @weshaggard's logic for building the meta-package pulls the full version from the project.json.  In that case we'd get stable versions just by updating that project.json when we ingest a new build.